third-party/tcmalloc/Makefile

```diff
@@ -5,20 +5,29 @@ endif
 CHPL_MAKE_HOST_TARGET = --target
 include $(CHPL_MAKE_HOME)/make/Makefile.base
 
+#
+# Cray X* builds are cross-compilations.
+#
+ifneq (, $(filter cray-x%,$(CHPL_MAKE_TARGET_PLATFORM)))
+CHPL_TCMALLOC_CFG_OPTIONS += --host=x86_64-cle-linux-gnu
+endif
+
 # enable minimal and frame pointers??
-CONFIG_FLAGS = --config-cache \
-               --enable-shared=no \
-               --enable-minimal \
-               --disable-libc-malloc-override \
-               --prefix=$(TCMALLOC_INSTALL_DIR)
+CHPL_TCMALLOC_CFG_OPTIONS += --config-cache \
+                             --enable-shared=no \
+                             --enable-minimal \
+                             --disable-libc-malloc-override \
+                             --prefix=$(TCMALLOC_INSTALL_DIR)
 
 ifneq (, $(filter cray-%,$(CHPL_MAKE_TARGET_PLATFORM)))
 # On some Cray systems we need to bring in libpthread explicitly when
 # building internal tests, to satisfy references from other packages that
 # don't do it themselves.  This is expected to be temporary.
-CONFIG_FLAGS += LDFLAGS='$(LDFLAGS) -lpthread'
+CHPL_TCMALLOC_CFG_OPTIONS += LDFLAGS='$(LDFLAGS) -lpthread'
 endif
 
+CHPL_TCMALLOC_CFG_OPTIONS += $(CHPL_TCMALLOC_MORE_CFG_OPTIONS)
```

Comment:
I don't see CHPL_TCMALLOC_MORE_CFG_OPTIONS defined anywhere. Is that also intentional?